python/src/sdf/pyJoint.cc

```diff
@@ -34,6 +36,7 @@ namespace python
 void defineJoint(pybind11::object module)
 {
   pybind11::class_<sdf::Joint> jointModule(module, "Joint");
+  GZ_UTILS_WARN_IGNORE__DEPRECATED_DECLARATION
```

Comment:
Should we even include the deprecated functions in the python API since we don't have a stable release of the bindings yet?

Reply:
gz-mujoco is already using the deprecated functions, so I was going to leave them in until gz-mujoco can be updated to avoid breakage. Should I just remove it and then quickly fix it in a gz-mujoco PR?

Reply:
Yeah, that sounds good to me.

Reply:
removed deprecated API from python binding in https://github.com/gazebosim/sdformat/pull/1053/commits/70023d623a8759be21804c08d71d8943aeeda113